utils/xbfmg.c

```diff
@@ -17,20 +26,14 @@
 
 void prtusage( char * );
 
-#define MXFLEN 125
+#define MXFLEN 125 /**< ??? */
```

Comment:
@jbathegit what is this?

Reply:
I'll fill this one in as well in my follow-on commit.

Reply:
done